### PR TITLE
Navigation updates

### PR DESCRIPTION
**Pull request by dashosaur**
- Make a generic navigation section to go at the top of each page. I added some css for format into a horizontal nav bar that we can make prettier later. 
- Create a category page for each book club that links to each study guide and can have a custom description. 
- Separate out the header content into an include so we don’t duplicate code.

### CODE REVIEW

**Review by lizgene — CHANGES_REQUESTED** [review]
I think we can make this a bit more dynamic by using the `site.categories` variable to generate items in the nav bar and links to each category page, plus a specific `category` layout that can be re-used. 

Example using `site.categories`:

```liquid
{% for category in site.categories %}
  <li><a name="{{ category | first }}">{{ category | first }}</a>
    <ul>
    {% for posts in category %}
      {% for post in posts %}
        <li><a href="{{ post.url }}">{{ post.title }}</a></li>
      {% endfor %}
    {% endfor %}
    </ul>
  </li>
{% endfor %}
```

Also calling @pamo for some SASS! :)  Let's at least get this horizontal before merging. I won't be able to work on this tonight but can come back for reviews. 

image

**Comment by dashosaur** [comment]
@lizhubertz I intentionally made the navigation list hardcoded so that we could use other categories elsewhere and not have them show up in the nav bar. I agree that we should have a category layout but I'd like to do that in a separate patch. Where were you suggesting using that block of code exactly?

**Comment by lizgene** [comment]
@dashland There's also the concept of `tags` we can use for more non-navbar granularity, but also I won't be a blocker if you want to keep working. 👍  🚢 

Another thing you can do if you want to branch off changes you are working on (not block your work while making improvements to a specific feature // make sure pull requests are not SO big they become ridiculous) is do a `git branch` directly from your working branch. Then once your original branch is merged into `master` you can do a rebase or just `git pull origin master` to merge in those changes // only your new branch's changes will show up in your diff. :-)

**Review by lizgene — APPROVED** [review]

File: _sass/_navigation.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    box-shadow: none;
+    text-shadow: none;
+    display: block;
+    padding: 10px 20px;
+}
+
+nav ul li {
+    background: $menu-background;
+    position: relative;
+    float: left;
```
Comment (pamo) [inline _sass/_navigation.scss:23]: no 🙅🏻
Let's use a [flexbox approach](https://teamtreehouse.com/library/css-flexbox-layout/building-a-layout-with-flexbox/building-a-navigation-bar-with-flexbox) please.

**Review by pamo — CHANGES_REQUESTED** [review]

File: _sass/_navigation.scss (hunk 1 above)
Comment (dashosaur) [reply _sass/_navigation.scss:23]: hehe, will do! (I haven't done web dev in a decade so I don't know any of the new stuff yet!)

**Comment by dashosaur** [comment]
thanks @lizhubertz! I am using local branches, but I won't get a chance to work on this very often so I would rather merge incremental changes as long as they improve the site and don't break things.

I will update the SASS to make this shipable!

**Pull request closed without merging**